source/chapters/appendix/mixxx_controls.rst

```diff
@@ -2506,6 +2506,19 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
 
    .. versionadded:: 2.0.0
 
+.. mixxx:control:: [ChannelN],update_replaygain_from_pregain
+                   [PreviewDeckN],update_replaygain_from_pregain
+                   [SamplerN],update_replaygain_from_pregain
+
+  Applies the deck pregain knob value to the detected ReplayGain value for the
+  current track. This is a way to update the ReplayGain value of a track if it
+  has been detected incorrectly. When this option is selected, the pregain value
+  for the deck will be adjusted so that there is no audible difference in track
+  volume, so this operation is safe to use during performance.
```

Comment:
```suggestion
  has been detected incorrectly. When this control is triggered, the pregain value
  for the deck will be centered so that there is no audible difference in track
  volume, so this operation is safe to use during performance.
```

Reply:
this is not quite true -- the deck that was the source of the adjustment will end up at the center position, but if there are any other decks with the track loaded they may not be adjusted to the center.

Reply:
But for the sake of simplicity, maybe it's fine to elide that detail :)

Reply:
applied manually

Reply:
ah, good point.